junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/descriptor/MethodTestDescriptor.java

```diff
@@ -108,27 +109,31 @@ public SkipResult shouldBeSkipped(JupiterEngineExecutionContext context) throws
 	@Override
 	public JupiterEngineExecutionContext execute(JupiterEngineExecutionContext context,
 			DynamicTestExecutor dynamicTestExecutor) throws Exception {
+		invokeBetweenBeforeAndAfter(context, c -> invokeTestMethod(c, dynamicTestExecutor));
+		return context;
+	}
+
+	protected void invokeBetweenBeforeAndAfter(JupiterEngineExecutionContext context,
+			Consumer<JupiterEngineExecutionContext> action) {
 		ThrowableCollector throwableCollector = context.getThrowableCollector();
 
 		// @formatter:off
 		invokeBeforeEachCallbacks(context);
```

Comment:
Please don't mess with the _indentation art_.

In other words, please retain the original indentation so that diffs are easier to follow.